components/team-codes/Create.js

```diff
@@ -20,84 +14,63 @@ export function Create(props) {
   const placeholderImage = require("../../assets/images/40by40.png");
 
   return (
-    <View style={styles.background}>
-      <View style={styles.main}>
+    <KitBackgroundScreen title="Set Team Name:">
+      <View style={{ flex: 2, justifyContent: "center" }}>
+        {/* Evan TODO: Make this input a separate compo */}
+        <TextInput
+          style={styles.textInput}
+          placeholder="Ex) Team One"
+          onChangeText={setTeamName}
+          value={teamName}
+        />
+      </View>
+
+      <View style={{ flex: 4, justifyContent: "flex-start" }}>
         <KitButton
-          style={{
-            maxHeight: 40,
-            marginLeft: 0,
-            marginRight: 0,
-            justifyContent: "flex-start"
+          style={{ button: styles.button }}
+          onPress={async () => {
+            const teamCode = await createTeam(teamName, "TODO: add userid");
```

Comment:
Shouldn't be too problematic, but I'm not sure if an `async` onPress callback is the best practice. 

Consider using `.then()` instead of `await` for the call to `createTeam`?